rules/import/on.js

```diff
@@ -46,6 +46,6 @@ module.exports = {
     //Enforce a newline after import statements
     "import/newline-after-import": 0,
     //Prefer a default export if module exports a single name
-    "import/prefer-default-export": 0,
+    "import/prefer-default-export": 0
```

Comment:
Wouldn't that make this is an error that caused linting (and therefore CI) to fail. If so, this isn't release prep either.